templates/shrine/src/shrine-app.html

```diff
@@ -356,13 +368,16 @@
     },
```

Comment:
In https://github.com/PolymerElements/app-layout/pull/344/files#diff-556771f4f0d01d7dcc6be44b98ff25eeR358, let's move it to attached.